doc/releases/changelog-dev.md

```diff
@@ -159,6 +159,9 @@
   The code that checks for qubit wise commuting (QWC) got a performance boost that is noticable
   when many commuting paulis of the same type are measured.
 
+* A `Sum` arithmetic class is added that allows users to represent the sum of operators
```

Comment:
Might be good to move the changelog entry to the `**Operator Arithmetic:**` section below so all the relevant PR's are grouped together.

Reply:
👍🏼